tests/TopDownProteomics.Tests/ProFormaWriterTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using NUnit.Framework;
+using TopDownProteomics.ProForma;
+
+namespace TestProject1
+{
+    [TestFixture]
+    public class ProFormaWriterTests
+    {
+        public static ProFormaWriter _writer = new ProFormaWriter();
+
+        [Test]
+        public void WriteSequenceOnly()
+        {
+            var term = new ProFormaTerm("SEQUENCE", null, null, null);
+            var result = _writer.WriteString(term);
+
+            Assert.AreEqual(term.Sequence, result);
+        }
+
+        [Test]
+        public void WriteSingleTag()
+        {
+            var term = new ProFormaTerm("SEQUENCE", null, null, new[]
+            {
+                new ProFormaTag(2, new[] { new ProFormaDescriptor("info", "test") })
```

Comment:
We should rename the first parameter and the property it sets, `Index` to `ZeroBasedIndex`.